SPECS-SIGNED/fwctl-signed/fwctl-signed.spec

```diff
@@ -0,0 +1,142 @@
+%{!?_release: %define _release OFED.24.10.0.6.7.1}
+
+%global target_kernel_version_full %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-headers))
+%global target_azurelinux_build_kernel_version %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}' $(/bin/rpm -q --whatprovides kernel-headers))
+%global target_kernel_release %(/bin/rpm -q --queryformat '%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-headers) | /bin/cut -d . -f 1)
+
+%global KVERSION %{target_kernel_version_full}
+%global K_SRC /lib/modules/%{target_kernel_version_full}/build
+
+# KMP is disabled by default
+%{!?KMP: %global KMP 0}
+
+%{!?KVERSION: %global KVERSION %{target_kernel_version_full}}
+%global kernel_version %{KVERSION}
+%global krelver %(echo -n %{KVERSION} | sed -e 's/-/_/g')
+# take path to kernel sources if provided, otherwise look in default location (for non KMP rpms).
+%{!?K_SRC: %global K_SRC /lib/modules/%{KVERSION}/build}
+
+# define release version
+%{!?src_release: %global src_release %{_release}_%{krelver}}
+%if "%{KMP}" != "1"
+%global _release1 %{src_release}
+%else
+%global _release1 %{_release}
+%endif
+%global _kmp_rel %{_release1}%{?_kmp_build_num}%{?_dist}
+
+Summary:	 %{_name} Driver
+Name:		 %{_name}
+Version:	 %{_version}
+Release:	 1%{?dist}
+License:	 GPLv2
+Url:		 http://nvidia.com
+Group:		 System Environment/Base
+
+# This package's "version" and "release" must reflect the unsigned version that
+# was signed.
+# An important consequence is that when making a change to this package, the
+# unsigned version/release must be increased to keep the two versions consistent.
+# Ideally though, this spec will not change much or at all, so the version will
+# just track the unsigned package's version/release.
+#
+# To populate these sources:
+#   1. Build the unsigned packages as normal
+#   2. Sign the desired binary
+#   3. Place the unsigned package and signed binary in this spec's folder
+#   4. Build this spec
+
+Source0:        %{name}-%{version}-%{release}.%{_arch}.rpm
+Source1:        fwctl.ko
+Source2:        mlx5_fwctl.ko
+
+BuildRoot:	/var/tmp/%{name}-%{version}-build
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+ExclusiveArch:  x86_64
+
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  kernel-devel = %{target_kernel_version_full}
+BuildRequires:  kernel-headers = %{target_kernel_version_full}
+BuildRequires:  binutils
+BuildRequires:  systemd
+BuildRequires:  kmod
+BuildRequires:  mlnx-ofa_kernel-devel = %{_version}
+BuildRequires:  mlnx-ofa_kernel-source = %{_version}
+
+Requires:       mlnx-ofa_kernel = %{_version}
+Requires:       mlnx-ofa_kernel-modules  = %{_version}
+Requires:       kernel = %{target_kernel_version_full}
+Requires:       kmod
+
+%description
+fwctl signed kernel modules
+
+%prep
+
+%build
+rpm2cpio %{Source0} | cpio -idmv -D %{buildroot}
+
+%install
+cp %{Source1} %{buildroot}/lib/modules/%{KVERSION}/updates/fwctl/fwctl.ko
```

Comment:
where is %license for this?

And...if we get to choose the license, we should be choosing BSD.